src/app/types.ts

```diff
@@ -0,0 +1,20 @@
+export type SubmitButtonProps = {
+  children: React.ReactNode;
+  className?: string;
```

Comment:
this is optimizing too early. We do not need this component to accept customization.

watch this for more context on this thinking https://www.deconstructconf.com/2019/dan-abramov-the-wet-codebase